config/settings.yml

```diff
@@ -1288,11 +1288,9 @@ vanotify:
       api_key: fake_secret
       template_id:
         appoint_a_representative_confirmation_email: appoint_a_representative_confirmation_email_template_id
-        burial_claim_confirmation_email_template_id: burial_claim_confirmation_email_template_id
         ch31_vbms_form_confirmation_email: ch31_vbms_fake_template_id
         ch31_central_mail_form_confirmation_email: ch31_central_mail_fake_template_id
         career_counseling_confirmation_email: career_counseling_confirmation_email_template_id
-        form527ez_confirmation_email: form527ez_confirmation_email_template_id
         preneeds_burial_form_email: preneeds_burial_form_email_template_id
```

Comment:
Will you also submit a PR to the manifest repo to remove these values from there?

Reply:
https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3181
https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3185